Marlin/src/HAL/HAL_STM32F1/msc_sd.cpp

```diff
@@ -0,0 +1,57 @@
+/**
+ * Marlin 3D Printer Firmware
+ *
+ * Copyright (c) 2019 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ * Copyright (c) 2019 BigTreeTech [https://github.com/bigtreetech]
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#include "../../inc/MarlinConfig.h"
+#include "msc_sd.h"
+#include "onboard_sd.h"
+#include "spi.h"
```

Comment:
On linux, headers are case sensitive... should be SPI.h

Reply:
It has been cleaned up.